Configuration/TCA/Overrides/tt_content.php

```diff
@@ -49,12 +49,12 @@
 );
 
 $GLOBALS['TCA']['tt_content']['palettes']['linklabel'] = [
```

Comment:
What's the best practice for palette keys, should they also go into the same namespace?

Reply:
i think this is not so important, because you can change it in our site-package, if you have conflicts (on the other hand:  you cannot change DB-Fields in site-packages)